README.md

```diff
@@ -1 +1,55 @@
-# Capstone-Attempt-1
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# DL Global Summit
+
+> In this project I have recreated one of Cindy Shin's (link in the acknowledgments section) pages but with my own content. The two pages recreated are the home page and the about page. The recreation is made to be similar to the original design but not exact.
+
+
+## Built With
+
+- HTML, SCSS, JavaScript
+
+## Live Demo 
+
+[Live Demo Link](https://dodoburner.github.io/DL-Global-Summit/)
+
+## Getting Started
+
+How to run this project locally:
```

Comment:
- [ ] you have added the header "how to run this project locally" it seems however, that you have not committed the steps requested by the previous CR, which might look something like below:
- use the `cd` command to go to the folder you want to save the project
- use the command `git clone https://github.com/dodoburner/DL-Global-Summit.git` to download the project.
and so forth...